src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -174,10 +174,31 @@
 // TODO: Overrides that need to be removed.
 
 .ms-Breadcrumb-overflow {
+  @include margin-right(0px);
```

Comment:
You shouldn’t be having overrides anymore since the style you’re overwriting is just above in the very same file…